src/frontend/src/lib/components/tokens/TokensSignedIn.svelte

```diff
@@ -21,10 +22,38 @@
 		({ id: tokenId }) =>
 			($balancesStore?.[tokenId]?.data ?? BigNumber.from(0n)).gt(0n) || displayZeroBalance
 	);
+
+	let sortingEnabled = true;
+	$: $pointerEventStore, (sortingEnabled = !$pointerEventStore);
+
+	const updateTokensToDisplay = ({
+		tokensToDisplay,
+		newTokensList
+	}: {
+		tokensToDisplay: Token[];
+		newTokensList: Token[];
+	}): Token[] => {
+		if (sortingEnabled) {
+			return newTokensList;
+		}
+
+		const tokenMap = new Map(
```

Comment:
I'm not sure I understand, you use a map to ensure the token are always sorted the same way?

Reply:
correct, this map would just keep the order, so that we can just update the balances (wrote some comments in the code)